M/MariaDB_Connector_C/build_tarballs.jl

```diff
@@ -3,13 +3,13 @@
 using BinaryBuilder
 
 name = "MariaDB_Connector_C"
-version = v"3.1.12"
+version = v"3.3.1"
```

Comment:
Looks like [v3.3.2](https://github.com/mariadb-corporation/mariadb-connector-c/releases/tag/v3.3.2) was released in the meantime, do you want to upgrade?

Reply:
Yeah, that'd be great! Thanks so much for helping here!